parsing/lexer.mll

```diff
@@ -330,29 +315,21 @@ rule token = parse
       { UIDENT(Lexing.lexeme lexbuf) }       (* No capitalized keywords *)
   | uppercase_latin1 identchar_latin1 *
       { warn_latin1 lexbuf; UIDENT(Lexing.lexeme lexbuf) }
-  | int_literal
-      { try
-          INT (cvt_int_literal (Lexing.lexeme lexbuf))
-        with Failure _ ->
-          raise (Error(Literal_overflow "int", Location.curr lexbuf))
+  | int_literal { INT (Lexing.lexeme lexbuf, None) }
+  | int_literal literal_modifier
+      {
```

Comment:
This would really benefit from using ocamllex's named sub-strings:

| (int_literal as lit) (literal_modifier as modif) { INT (lit, Some modif) }